src/components/app/Popouts/Popouts/AddServer.vue

```diff
@@ -141,6 +141,10 @@ export default {
         this.closeMenu();
       }
     },
+    keyDownEvent(event) {
+      if (event.keyCode !== 27) return; // 27 = escape
```

Comment:
`event.keyCode` is [deprecated](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/keyCode), changing this to `event.key !== "Escape"` would be better.